component/common/relabel/relabel_test.go

```diff
@@ -75,6 +75,8 @@ func TestParseConfig(t *testing.T) {
 			t.Parallel()
 			var cfg Config
 			err := river.Unmarshal([]byte(tt.cfg), &cfg)
+			cfg.SetToDefault()
+			cfg.Validate()
```

Comment:
I'm not sure what's going on here; why is SetToDefault being called after Unmarshal, and why is the error of cfg.Validate() not checked?

Reply:
I think the technical term is a 'brain fart'